File: atividade_C/atividade10.c
#include <stdio.h>

int idade,menor_idade,maior_idade,qtd_mulheres;
float salario;
char sexo;  

void VerificarIdades(int idade_temp);
void VerificarSalarioFem(void);

int main()
{
    float media_salario = 0.0;
    
    menor_idade = 0;
    maior_idade = 0;
    
    for(int i=1; i<=4; i++)
    {
        printf("Informe sua idade: ");
        scanf("%d", &idade);
        printf("\nInforme seu sexo [M] ou [F]: ");
        scanf("%s", &sexo);
        printf("\nInforme seu salario: ");
        scanf("%f", &salario);

        printf("\n================\n");
        media_salario = media_salario + salario;

        VerificarIdades(idade);
        VerificarSalarioFem();
    }

    media_salario = media_salario/4;
    printf("Média de salário do grupo: R$ %.2f\n", media_salario);
    printf("Maior idade: %d Menor idade: %d \n",maior_idade,menor_idade);
    printf("Mulheres com salário até R$1.500,00: %d",qtd_mulheres);

    return 0;
}

void VerificarSalarioFem()
{
    switch(toupper(sexo))
    {
        case 'F':
            if(salario <= 1500.00)
            {
              qtd_mulheres++;
            }

            break;
    }
}

void VerificarIdades(int idade_temp)
{

    if(menor_idade > idade_temp || menor_idade == 0)
    {
            menor_idade = idade_temp;
    }
    else if(maior_idade < idade_temp || maior_idade == 0)
    {
            maior_idade = idade_temp;
    }
}

File: 2024/atividade_py/atividade10.py
def VerificarIdades(idade_temp):
    global menor_idade,maior_idade;

    if menor_idade > idade_temp or menor_idade == 0:
        menor_idade = idade_temp;
    elif maior_idade < idade_temp or maior_idade ==0:
        maior_idade = idade_temp;

def VerificarSalarioFem(sexo):
    global qtd_mulheres;
    match sexo:
        case 'f':
            if salario <= 1500.00:
                qtd_mulheres+= 1;

media_salario = 0.0;
menor_idade = maior_idade = qtd_mulheres = tipo = i = 0;

while tipo == 0:
    print("Informe sua idade: ", end="");
    idade = int(input());
    print("Informe seu sexo [M] ou [F]: ",end="");
    sexo = input();
    print("Informe seu salario: ",end="");
    salario = float(input());

    print("==================");
    media_salario = media_salario + salario;

    VerificarIdades(idade);
    VerificarSalarioFem(sexo); 

    i+= 1;

    print("Deseja continuar ?\n[0] - Sim\n[1] - Não\nResposta: ", end="");
    tipo = int(input());

media_salario = media_salario/i;

print(f"Média de salário do grupo: R$ {media_salario:.2f}");
print(f"Maior idade: {maior_idade} Menor idade: {menor_idade}");
print(f"Mulheres com salário até R$1.500,00: {qtd_mulheres}");



